modules/tinymce/src/core/main/ts/commands/FontCommands.ts

```diff
@@ -58,23 +45,33 @@ const normalizeFontNames = (font: string) => {
   }).join(',');
 };
 
+const fontQuery = (editor: Editor, fontProp: FontInfo.FontProp) => {
+  const getter = fontProp === 'font-family' ? FontInfo.getFontFamily : FontInfo.getFontSize;
+  const selection = editor.selection.getRng().cloneRange();
```

Comment:
This name is rather confusing, as this isn't a `Selection` object, it's a `Range` object. I'd suggest calling this `rng` or `range`.